src/utils/CoinbasePixel.ts

```diff
@@ -32,7 +37,7 @@ export type CoinbasePixelConstructorParams = {
   host?: string;
   appId: string;
   appParams: JsonObject;
-  onReady?: (error?: Error) => void;
+  onReady?: (error?: OnReadyError) => void;
```

Comment:
@lachiet Should this be made required at this point?

Reply:
Yep makes sense.